extensions/git/src/git.ts

```diff
@@ -1913,6 +1917,11 @@ export class Repository {
 		}
 	}
 
+	async getBranches(query: BranchQuery): Promise<Ref[]> {
```

Comment:
You can still return `string[]` if you want. 👍

Reply:
Might as well return refs, since they have more info. It is also better for when using the `remote` query.